.github/readmes/javascript/_setup-2.md

```diff
@@ -20,6 +20,27 @@ prisma2 lift up
 
 You can now use the [SQLite Browser](https://sqlitebrowser.org/) to view and edit your data in the `./prisma/dev.db` file that was created when you ran `prisma2 lift up`.
 
+<Details>
+<Summary><b>Alternative: </b>Connect to your own database</Summary>
+
+Prisma supports [other databases](https://github.com/prisma/prisma2/blob/master/docs/supported-databases.md). If you would like to connect to your own database, you can do so by specifying a different data source in the [Prisma schema file](prisma/schema.prisma):
+
+```
+datasource db {
+    provider = "DATABASE_PROVIDER"
+    url      = "DATABASE_PROVIDER://USERNAME:PASSWORD@HOST:PORT/DATABASE?schema=SCHEMA_FILE_NAME"
+    default  = true
+}
+```
+
```

Comment:
I'd suggest to also add a sample `datasource` configuration that shows a concrete example for PostgreSQL or MySQL, e.g.:

```prisma
datasource postgresql {
  provider = "postgresql"
  url      = "postgresql://johndoe:secret42@localhost:5432"
```

Reply:
Should I show an example for each...or really just pick one?

Reply:
Moreover, don't I have to specify a database name in the connection string for both MySQL and PostgreSQL?

Reply:
> Should I show an example for each...or really just pick one?

Actually, as we only support two other DBs right now let's just throw in both.

> Moreover, don't I have to specify a database name in the connection string for both MySQL and PostgreSQL?

 Ah yes, that's required for both I think! :)